node-notes-cli/app.js

```diff
@@ -0,0 +1,29 @@
+const dataJSON = require('./data.json');
+const fs = require('fs');
+
+if (process.argv[2] === 'read') {
+  fs.readFile('./data.json', 'utf8', err => {
+    if (err) throw (err);
```

Comment:
You can completely skip this `readFile` step if you are using `require()` to load the data.